docs/src/content/docs/ja/components/using-components.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: コンポーネントの使用
+description: StarlightでMDXとMarkdocにコンポーネントを使用する。
+sidebar:
+  order: 1
+---
+
+コンポーネントを使用すると、UIの一部やスタイリングを一貫して簡単に再利用できます。
+例えば、リンクカードやYouTube埋め込みなどが挙げられます。
+Starlightは[MDX](https://mdxjs.com/)と[Markdoc](https://markdoc.dev/)ファイルでのコンポーネントの使用をサポートし、いくつかの一般的なコンポーネントを提供しています。
+[Astro Docsでコンポーネントの構築についてさらに学ぶ](https://docs.astro.build/ja/basics/astro-components/)。
+
+## MDXでコンポーネントを使用する
+
+コンポーネントを使用するには、MDXファイルにインポートし、JSXタグとしてレンダリングします。
+これらはHTMLタグのように見えますが、`import`文の名前に一致する大文字で始まります：
+
+```mdx
+---
+# src/content/docs/example.mdx
+title: Welcome to my docs
+---
+
+import { Icon } from '@astrojs/starlight/components';
+import CustomCard from '../../components/CustomCard.astro';
+
+<Icon name="open-book" />
+
+<CustomCard>
+	コンポーネントには**ネストされたコンテンツ**も含めることができます。
+</CustomCard>
+```
+
+StarlightはAstroを基盤としているため、MDXファイルで[サポートされているUIフレームワーク（React、Preact、Svelte、Vue、Solid、Alpine）](https://docs.astro.build/en/core-concepts/framework-components/)で構築されたコンポーネントのサポートを追加できます。
+Astroドキュメントの[MDXでのコンポーネントの使用](https://docs.astro.build/en/guides/integrations-guide/mdx/#using-components-in-mdx)についてさらに学んでください。
+
+## Markdocでコンポーネントを使用する
+
+[Markdocセットアップガイド](/ja/guides/authoring-content/#markdoc)に従って、Markdocでコンテンツを作成するためのサポートを追加してください。
+
+Starlight Markdocプリセットを使用すると、Markdocの`{% %}`タグ構文でStarlightの組み込みコンポーネントを使用できます。
+MDXとは異なり、Markdocのコンポーネントはインポートする必要がありません。
+以下の例は、MarkdocファイルでのStarlightの[カードコンポーネント](/ja/components/cards/)のレンダリングを示しています：
+
+```markdoc
+---
+# src/content/docs/example.mdoc
+title: Welcome to my docs
+---
+
+{% card title="Stars" icon="star" %}
```

Comment:
Same here for the `title`.

Reply:
[b66afbd](https://github.com/withastro/starlight/pull/2384/commits/b66afbd3848dc46ca809d4b34c0bb717e8324391)

fixed here!